examples/onnxrt/nlp/huggingface_model/code_detection/quantization/ptq_dynamic/README.md

```diff
@@ -0,0 +1,54 @@
+Step-by-Step
+============
+
+This example quantizes the [microsoft/codebert-base](https://huggingface.co/microsoft/codebert-base) fine-tuned on the the [code defect detection](https://github.com/microsoft/CodeXGLUE/tree/main/Code-Code/Defect-detection#codexglue----defect-detection) task.
+
+# Prerequisite
+
+## 1. Environment
+```shell
+pip install neural-compressor
+pip install -r requirements.txt
+```
+> Note: Validated ONNX Runtime [Version](/docs/source/installation_guide.md#validated-software-environment).
+
+
+## 2. Prepare Dataset
+Run `prepare_data.sh` script to download dataset from website to `dataset` folder and pre-process it:
+
+```shell
+bash prepare_data.sh
+```
+## 3. Prepare Model
```

Comment:
Please update the Prepare Model step to `python prepare_model.py --input_model=a --output_model=b` format

Reply:
Will update it with another PR after the model upload and `prepare_model.py` template are finalized :)